scripts/build/ExternalAssemblyVersions.cs

```diff
@@ -0,0 +1,12 @@
+namespace Microsoft.VisualStudio.TestPlatform
+{
+    internal class ExternalAssemblyVersions
+    {
+        /// <summary>
+        /// Refers to the versions of the assemblies retrieved from the 
+        /// Microsoft.QualityTools.Testing.Fakes.TestRunnerHarness package.
+        /// The Package version can be found in "scripts\build\TestPlatform.Dependencies.props"
+        /// </summary>
+        internal const string MicrosoftFakesAssemblyVersion = "17.0.0.0";
```

Comment:
Why not to add this version value inside `TestPlatform.Dependencies.props` file?

Reply:
This is so this can be referenced inside .cs files. Values from within `TestPlatform.Dependencies.props` cannot be referenved by .cs files.

For more context: There seems to be hardcoded versions of assemblies in many .cs files (refer to abhitej's comment above) and in instances like this where a bunch of them are moving from 16.0.0.0 to 17.0.0.0, it would be better to have a central file so the gaps can be caught earlier on.